File: ws/pastor/u1/ws06JSON/ec.espe.edu.chickenFarmSimulator.model/poop.js
class Poop{
    constructor(amount){
        this._size = amount;
    }

    toString(){
        return `Poop amount => ${this._amount}`;
    }

    setAmount(amount){
        this._amount = amount;
    }

    getAmount(){
        return this._amount;
    }
}

    module.export = Poop;

File: hw/pastor/u1/hw14SafeInJsonChickenFarmSimulator/ChickenFarmSimulatorJava/src/main/java/ec/espe/edu/chickenFarmSimulator/model/Poop.java
package ec.espe.edu.chickenFarmSimulator.model;
import java.util.Random;
/**
 *
 * @author Mathews Pastor
 */
public class Poop{
    private int amount;
    
    public Poop(){
        Random random = new Random();
        this.amount = random.nextInt(51);
    }

    @Override
    public String toString() {
        return "Poop amount => " + amount;
    }

    public int getAmount() {
        return amount;
    }

    public void setAmount(int amount) {
        this.amount = amount;
    }
}
